chia/util/condition_tools.py

```diff
@@ -25,12 +25,28 @@ def parse_sexp_to_condition(
     Takes a ChiaLisp sexp and returns a ConditionWithArgs.
     If it fails, returns an Error
     """
-    as_atoms = sexp.as_atom_list()
-    if len(as_atoms) < 1:
+    first = sexp.pair
```

Comment:
The name `first` is a bit misleading, since it's actually a pair. Really, `first` is `sexp.pair[0]`.